sys/linux/bpf_lsm.txt

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/linux/bpf.h>
+
+resource fd_bpf_lsm_prog[fd]
+resource btf_id[int32]
+
+bpf_lsm_func_names = "bpf_lsm_path_mkdir"
```

Comment:
Write one-line shell that extracts all of them from vmlinux and paste output at the bottom of this file + the script as a comment.